File: Tugas2/java/disk.java
public class disk{
    private String typed;
    private String capacity;
    private int priced;

    // construct
    public disk() {}

    public disk(String typed, String capacity, int priced){
        this.typed = typed;
        this.capacity = capacity;
        this.priced = priced;
    }
	
	//set and get
	public void settyped(String typed){
		this.typed = typed;
	}
	public String gettyped(){
		return this.typed;
	}
	
	public void setcapacity(String capacity){
		this.capacity = capacity;
	}
	public String getcapacity(){
		return this.capacity;
	}
	
	public void setpriced(int priced){
		this.priced = priced;
	}
	public int getpriced(){
		return this.priced;
	}

    public void printdisk(){
        System.out.println("Tipe : " + gettyped());
        System.out.println("Kapasitas : " + getcapacity());
		System.out.println("Harga : " + getpriced());
    }

}

File: Tugas2/php/disk.php
<?php

class disk
{
	
	private $type = "";
	private $capacity = "";
	private $priced = 0;
	
	//Construct
	public function __construct($type = "", $capacity = "", $priced = 0){
		$this->type = $type;
		$this->capacity = $capacity;
		$this->priced = $priced;
	}
	
	//Set dan get
	public function settype($type){
		$this->type = $type;
	}
	public function gettype(){
		return $this->type;
	}
	
	public function setcapacity($capacity){
		$this->capacity = $capacity;
	}
	public function getcapacity(){
		return $this->capacity;
	}
	
	public function setpriced($priced){
		$this->priced = $priced;
	}
	public function getpriced(){
		return $this->priced;
	}

    //Print
    public function printdisk(){
		echo "Tipe : ". $this->gettype(). "<br/>";
        echo "Kapasitas : ". $this->getcapacity(). "<br/>";
		echo "Harga : ". $this->getpriced(). "<br/>";
	}
}
?>